azure-pipelines.yml

```diff
@@ -16,13 +16,13 @@ strategy:
     # All configs use
     # DA ghc-lib https://digitalassetsdk.bintray.com/ghc-lib/ghc-lib-parser-8.8.1.20191129.tar.gz
     linux-ghc-lib-881-8103:
-      image: "Ubuntu 16.04"
+      image: "ubuntu-20.04"
```

Comment:
apparently CI hasn’t run for so long that the linux & macos builders don’t exist at all and the windows builder is dead in a month or so.